egi2bids/mff2bids.py

```diff
@@ -44,28 +50,71 @@
     "249", "250", "251", "F9", "253", "254", "255", "256",
     "Cz",
 ]
+
+
 # fmt: on
+def _extract_folder(file):
+    """Extracts a compressed folder to its original form"""
+    # Get the path and file extension
+    file = Path(file)
+    ext = file.suffix
+    # Open the archive file based on its type
+    print(file)
+    if ext == ".tar":
+        path = file.with_name(file.name.replace(".tar", ""))
```

Comment:
```suggestion
        path = file.with_name(file.name.replace(ext, ""))
```

I think this syntax is a bit safer as it relies on `ext = "".join(file.suffixes)` and reduces the amount of hard-coded variables.